_includes/js/custom.js

```diff
@@ -99,4 +69,383 @@ jQuery(document).ready(function() {
         }).debounce(100));
     };
     search();
+    //set the first character os string to uppercase
+    function capitalize(string) {
+        return string.charAt(0).toUpperCase() + string.slice(1);
+    }
+    //surround data table rows generated in rendertable function with table head
+    function tableHtml(content, domain = false, tableDomains) {
+        return `<div><strong>${ domain? capitalize(tableDomains): ""}</strong></div>
+                <table id="ont_table" class="table table-hover sortable">
+                    <thead>
+                        <tr class="row">
```

Comment:
Inconsistent vertical alignment of table headers is jarring

<img width="1178" alt="Screen Shot 2022-07-31 at 14 56 53" src="https://user-images.githubusercontent.com/5069736/182027533-6537c559-6167-4a08-b0ea-4d5bda90d5f2.png">

Reply:
Agreed, added this to should fix list